app/src/main/java/com/example/a310_rondayview/MainActivity.java

```diff
@@ -1,14 +1,40 @@
 package com.example.a310_rondayview;
 
 import androidx.appcompat.app.AppCompatActivity;
+import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentManager;
+import androidx.fragment.app.FragmentTransaction;
 
 import android.os.Bundle;
 
+import com.example.a310_rondayview.databinding.ActivityMainBinding;
+
 public class MainActivity extends AppCompatActivity {
 
+    ActivityMainBinding binding;
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_main);
+        binding = ActivityMainBinding.inflate(getLayoutInflater());
+        setContentView(binding.getRoot());
+        replaceFragment(new CreateEventFragment());
+
+        binding.bottomNavigationView.setOnItemSelectedListener(item -> {
```

Comment:
Maybe add a small comment here saying this is where you link in the respective fragments